inc/post-templates.php

```diff
@@ -63,17 +65,17 @@ function post_templates_dropdown() {
  * Filter the single template value, and replace it with
  * the template chosen by the user, if they chose one.
  */
-if( !function_exists( 'get_post_template' ) ) {
+if( ! function_exists( 'get_post_template' ) ) {
 	function get_post_template( $template ) {
 		global $post;
 		if ( is_object( $post ) ) {
 			$custom_field = get_post_meta( $post->ID, '_wp_post_template', true );
 		}
 		if ( !empty( $custom_field ) ) {
-			if ( file_exists( get_stylesheet_directory() . "/{$custom_field}") ) {
-				$template = get_stylesheet_directory() . "/{$custom_field}";
-			} else if ( file_exists( get_template_directory() . "/{$custom_field}") ) {
-				$template = get_template_directory() . "/{$custom_field}";
+			if ( file_exists( get_stylesheet_directory() . '/{$custom_field}' ) ) {
```

Comment:
Tests failed to run because the branch was deleted before tests could be run: https://travis-ci.org/INN/Largo/builds/166745124